### PR TITLE
[Kotlin migration] CardTemplateBrowserAppearanceEditor

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Purpose / Description
I want kotlin be 50%+ of the repo

## Approach
Standard procedure

## How Has This Been Tested?

On my phone (Samsung Galaxy Note 10 Lite SM-N770F/DS, API 30)

https://user-images.githubusercontent.com/69634269/162092088-24f46016-b79c-475b-a5c9-109385153cdd.mp4

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
Didn't know this option existed, kind of hard to find

**Comment by BrayanDSO** [comment]
Just noticed this is another activity with title not being translated



I'll make the issue

**Review by mikehardy — APPROVED** [review]
Looks great to me, thanks!

**Comment by mikehardy** [comment]
Blocking #10748 so bumping review priority

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt [hunk 1]
```diff
@@ -13,251 +13,217 @@
...
+    private lateinit var mAnswerEditText: EditText
+    override fun onCreate(savedInstanceState: Bundle?) {
         if (showedActivityFailedScreen(savedInstanceState)) {
-            return;
+            return
         }
-        super.onCreate(savedInstanceState);
-        Bundle bundle = savedInstanceState;
+        super.onCreate(savedInstanceState)
+        var bundle = savedInstanceState
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt:45]: Cleanup: Use `?:`.

Comment (BrayanDSO) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt:46]: Done

**Review by mikehardy — APPROVED** [review]
still lgtm, pending ci

**Pull request merged**